### ISSUE TITLE
BarDropdownToggle missing aria-haspopup and aria-expanded attributes

### ISSUE THREAD

**Issue by stsrki** [Megabit/Blazorise#6599]
https://blazorise.com/support/issues/443/bardropdowntoggle-missing-aria-haspopup-and-aria-expanded-attributes
